src/dataviews/category-dataview/search-model.js

```diff
@@ -81,13 +81,13 @@ module.exports = Model.extend({
 
   _parseData: function (categories) {
     var newData = [];
-    _.each(categories, function (d) {
-      if (!d.agg) {
+    _.each(categories, function (data) {
+      if (!data.agg) {
         newData.push({
           selected: false,
-          name: (d.category || d.name).toString(),
-          agg: d.agg,
-          value: d.value
+          name: data.category || data.name || '',
```

Comment:
Perhaps it doesn't happen but...

The original code implies that `category` or `name` can be something apart from a string, a number for instance. Then, it converts it to string.

In your change, if `category` or `name` comes as numbers, they'll be passed along.

`(data.category || data.name || '').toString()` would do the trick, right?